src/components/TextField/TextField.scss

```
@@ -209,7 +209,7 @@ $stacking-order: (
   position: relative;
   z-index: z-index(contents, $stacking-order);
   flex: 0 0 auto;
-  color: currentColor;
+  color: var(--p-text-subdued);
```

```suggestion
  color: var(--p-text-subdued, currentColor);
```

Do we want to fallback to current color here for the old design language?

---

I think if this token doesn't exist the browser just renders black as the color